### DISCUSSION TITLE
Proposals for new standards and supplements

### DISCUSSION THREAD

**Discussion by drpaulralph** [acmsigsoft/EmpiricalStandards#23] [Ideas]
What additional standards or supplements should we create? Below are suggestions that have been raised already. Discussion of these and proposals for new stuff both welcome. 

### Standards

- Focus groups
- Discourse Analysis (e.g. studying StackOverflow)
- Cohort studies (not currently covered by the longitudinal studies standard)
- Questionnaires with predominately open-ended questions
- Repository mining that isn't a longitudinal study or data science (I still don't know what to do about such studies)
- Qualitative simulation like protocol studies and usability studies (I've opened an issue for this one). 
- Non-empirical manuscripts (this could be a supplement or a standard or just some kind of statement; I'm not sure yet)
- Novel or other, i.e., recommendations for reviewing a method for which there is no existing standard. (I've opened an issue for this one). 

### Supplements

- Replications 
- Theory Development

**Comment by ProfDoof** [comment]
Hi! I was looking at the Qualitative Surveys standard and noticed that it says that you should use the Discourse Analysis standard but that does not seem to exist. 

> If researchers collect written text or conversations (e.g. StackExchange threads), consider the Discourse Analysis Standard.

Until it is completed, it probably should not be referenced in the other standards.

**Reply by drpaulralph** [reply]
Good catch thanks.
